### ISSUE TITLE
Dialog seen for telemetry on embedded pages

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#3056]
Possibly due to automated password configuration that does not go through the setup screen.

![image](https://user-images.githubusercontent.com/1955897/119192896-b9253f80-ba78-11eb-91ac-e9ead5203105.png)

**Comment by vincent99** [comment]
Yes this was to capture users who had upgraded from a version that didn't initially have Telemetry (in maybe v1.2, ≈2016).  It came along for the ride in 2.x because it started from the same codebase, but I don't think there's any need to port this to Vue.

**Comment by nwmac** [comment]
@vincent99 Okay - I'll turn this off in Ember when it is embedded

**Comment by thehejik** [comment]
**Test report:**
Works as expected on current `rancher/rancher:master-head 5d49b92`.

* I did setup in Ember/Settings/Advanced -> option `telemetry-opt` to `prompt` value
* then I opened a new tab https://rancher/login and the telemetry dialog appeared  immediately after login in Ember UI
* then without confirming the dialog in Ember I opened another tab https://rancher to open Explorer UI
* and finally I tried in Explorer UI to create a new ec2 rke1 cluster by using Ember iframe and the dialog correctly didn't appear there.

Close.
